pkgs/data/fonts/shabnam-fonts/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "shabnam-fonts";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "rastikerdar";
+    repo = "shabnam-font";
+    rev = "v${version}";
+    sha256 = "1nya9cbbs6sgv2w3zyah3lb1kqylf922q3fazh4l7bi6zgm8q680";
```

Comment:
Doesn't build for me:
```
fixed-output derivation produced path '/nix/store/n6p39h3cs1ilrf6ngmjvfj5qhr7kwcyn-source' with sha256 hash '1y4w16if2y12028b9vyc5l5c5bvcglhxacv380ixb8fcc4hfakmb' instead of the expected hash '1nya9cbbs6sgv2w3zyah3lb1kqylf922q3fazh4l7bi6zgm8q680'
```

Reply:
Done!